src/style.css

```diff
@@ -1 +1,84 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(../src/fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: AvenirBook;
+  src: url(../src/fonts/Avenir-Book.ttf) format('truetype');
+  font-weight: 300;
+}
+
+input{
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  background: #FFF;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  color: #3D4E61;
+  font-size: 14px;
+}
+
+.input::placeholder {
+  font-weight: 300;
+  color: inherit;
+}
+
+.input_one input {
+  /* padding: 24px 0 23px 62px; */
+  font-family: AvenirBook, sans-serif;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.input_two, .input_one {
```

Comment:
The classes .input_two and .input_one are applied the same style. Consider merging them into one class to reduce code duplication.

Reply:
not fixed